### PR TITLE
Add Education Edit

### PR DESCRIPTION
**Pull request by cubeydice**
Resolves issue #4 

Description:
- Dependent upon changes in PR #18 
- Adds an EducationEdit component
- Allows users to edit an existing education by opening an edit form in a modal
- Submits a PATCH request to the /resume/education endpoint
- Added Edit icon to Education View

Changes Made:
- Created EducationView.jsx and EducationView.css
- Sends a PATCH request with the form data to the server

Future Updates:
- Use redux to manage application state

image
image

### CODE REVIEW

**Comment by cubeydice** [comment]
@Pradyuman7 This PR is ready for your review

File: package.json [hunk 1]
```diff
@@ -2,12 +2,14 @@
   "name": "orientation-project-js",
   "version": "0.1.0",
   "private": true,
+  "proxy":"http://localhost:5000/",
```
Comment (Pradyuman7) [inline package.json]: Let's remove this stuff from here as well

**Review by Pradyuman7 — APPROVED** [review]
LGTM 👍 

Can you also resolve the conflicts for this one, and then we can get it merged

File: package.json (hunk 1 above)
Comment (cubeydice) [reply package.json]: Proxy removed!

**Comment by cubeydice** [comment]
@Pradyuman7 
Conflicts resolved!

**Pull request merged**